samples/inputs/dropdown/target/src/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import { IgrDropdown, IgrButton, IgrDropdownItem, IgrDropdownModule, IgrButtonModule, IgrDropdownItemModule } from 'igniteui-react';
+import 'igniteui-webcomponents/themes/light/bootstrap.css';
+
+IgrDropdownModule.register();
+IgrDropdownItemModule.register();
+IgrButtonModule.register();
+
+export default class DropDownTarget extends React.Component<any, any> {
```

Comment:
Shouldn't this sample behave in a similar manner to the one in Web Components? ([this one](https://www.infragistics.com/webcomponents-demos/inputs/dropdown/target))

Reply:
The sample has been modified as the one in Web Components [here](https://www.infragistics.com/webcomponents-demos/inputs/dropdown/target).